src/Microsoft.DotNet.Cli.Utils/CommandResolution/IPlatformCommandSpecFactory.cs

```diff
@@ -0,0 +1,23 @@
+using System;
```

Comment:
The implementations of this do differ based on platform. For example on windows we need to wrap the command after it goes through the resolver with cmd.exe if it is a .cmd file.